recipes/nbterm/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "nbterm" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: abe1d971888c608888e6ade60bb61302cfc0d9dd021d8fd566b12310c5ee6a16
+
+build:
+  number: 0
+  entry_points:
+    - nbterm = nbterm.nbterm:cli
+  skip: true  # [py<37]
```

Comment:
can this not be noarch?

Reply:
I started like that, but it complained on Windows because nbterm is an executable.